### PR TITLE
Opt In accounts support and offline transactions broadcast

### PR DESCRIPTION
**Pull request by Peersyst**
- Added support for OptIn accounts with old curve
- Broadcast offline transactions by scanning QR code feature
- Fixed harvesting not asking unlocked accounts properly
- Fixed custom node not allowing .network dns names
- Fixed opt in update (need to test)

### CODE REVIEW

File: src/components/settings/SettingsNodeSelector.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ class SettingsNodeSelector extends Component {
 	};
 
 	isValidNode = (node) => {
-		return /^(https?:\/\/)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)$/.test(node);
+		return /^(https?:\/\/)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,8}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)$/.test(node);
```
Comment (OlegMakarenko) [inline src/components/settings/SettingsNodeSelector.js:132]: Will not work with custom domains

File: src/store/account.js [hunk 2]
```diff
@@ -60,14 +60,12 @@ export default {
                 if (state.account.refreshingObs) {
                     state.account.refreshingObs.unsubscribe();
                 }
+                await dispatchAction({type: 'account/loadBalance'});
+                dispatchAction({type: 'harvesting/init'});
+                dispatchAction({type: 'account/loadCosignatoryOf'});
                 const refreshingObs = from(
                     new Promise(async (resolve, reject) => {
```
Comment (OlegMakarenko) [inline src/store/account.js:67]: Do we need this Promise?

File: src/store/network.js [hunk 3]
```diff
@@ -106,14 +106,13 @@ export default {
                 try {
                     isUp = await NetworkService.isNetworkUp(selectedNetwork);
                 } catch {
-                    isUp = false;
+                    isUp = state.network.nodeFailedAttempts < 5;
```
Comment (OlegMakarenko) [inline src/store/network.js:109]: IMO better to store `5` in config

Comment (Peersyst) [reply src/store/network.js]: Yep, thanks putting that in a global variable. Will maybe move that into an env file in the future

File: src/store/account.js (hunk 2 above)
Comment (Peersyst) [reply src/store/account.js]: Deleting that!

File: src/components/settings/SettingsNodeSelector.js (hunk 1 above)
Comment (Peersyst) [reply src/components/settings/SettingsNodeSelector.js]: You are right, let's remove this restriction

**Review by OlegMakarenko — APPROVED** [review]

**Pull request merged**